src/Lykke.Service.LiteCoin.API/Controllers/BalancesController.cs

```diff
@@ -82,21 +85,30 @@ public BalancesController(IAddressValidator addressValidator, IWalletBalanceServ
         [SwaggerOperation(nameof(GetBalances))]
         [ProducesResponseType(typeof(PaginationResponse<WalletBalanceContract>), (int)HttpStatusCode.OK)]
         [ProducesResponseType(typeof(ErrorResponse), 400)]
-        public async Task<PaginationResponse<WalletBalanceContract>> GetBalances([FromQuery]int take, [FromQuery] string continuation)
+        public async Task<IActionResult> GetBalances([FromQuery] int take, [FromQuery] string continuation)
         {
             if (take <= 0)
```

Comment:
Please use same style for validation for all endpoints where there are take and continuation param. Like this: 

```
if (!ModelState.IsValid || 
    !ModelState.IsValidContinuationToken(continuation) || 
    !ModelState.IsValidTakeParameter(take))
{
    return BadRequest(ModelState.ToErrorResponce());
}
```